pilot/cmd/pilot-discovery/main.go

```diff
@@ -103,6 +103,8 @@ func init() {
 		"Enable profiling via web interface host:port/debug/pprof")
 	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discovery_cache", true,
 		"Enable caching discovery service responses")
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.WebhookEndpoint, "webhookEndpoint", "",
```

Comment:
Maybe have separate flags for lds/cds/rds webhooks ( and later for ldsv2, etc) ?

Reply:
No, I think we should just imitate ADS and combine all responses.